lib/ActivityStreams.js

```diff
@@ -60,6 +61,17 @@ function ActivityStreams(options = {}) {
   this._memoized = this._get_memoized(this._memoizer);
 
   this._telemetrySender = new TelemetrySender();
+
+  this._experimentData = {};
+  let rng = seedrandom(options.clientID);
+  // Targeting 10% of users for this experiment
```

Comment:
Hi! Just a curious Joe here. Did this code run on every new tab? If so, is this not putting 10% of new tab opens (as opposed to 10% of FF eligible FF installs) in the variant?

Reply:
Just saw the seedrandom(), so that should be ok :)